File: sample/sphere_sample/src/main/java/com/sphere/sample/SampleActivity.java
package com.sphere.sample;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class SampleActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sample);

        findViewById(R.id.sample_analytics).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(SampleActivity.this, SampleAnalyticsActivity.class));
            }
        });

        findViewById(R.id.sample_webview).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(SampleActivity.this, SampleWebViewActivity.class));
            }
        });

    }

}

File: sample/sphere_sample_kt/src/main/java/com/sphere/sample/kt/SampleActivity.kt
package com.sphere.sample.kt

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View

class SampleActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_sample)

        findViewById<View>(R.id.sample_analytics).setOnClickListener {
            startActivity(Intent(this, SampleAnalyticsActivity::class.java))
        }

        findViewById<View>(R.id.sample_webview).setOnClickListener {
            startActivity(Intent(this, SampleWebViewActivity::class.java))
        }
    }
}
